Formula/xcbuild.rb

```diff
@@ -0,0 +1,22 @@
+class Xcbuild < Formula
+  desc "Xcode-compatible build tool"
+  homepage "https://github.com/facebook/xcbuild"
+  url "https://github.com/facebook/xcbuild.git", :tag => "0.1.0"
+  head "https://github.com/facebook/xcbuild.git", :shallow => false
+
+  depends_on "ninja" => :recommended
+  depends_on "cmake" 
+  depends_on "libpng"
+  depends_on "libxml2"
```

Comment:
I assumed Homebrew would use the system one magically... My bad! I've removed it.